src/python-bindings/JobEventLog.cpp

```diff
@@ -140,7 +140,16 @@ JobEventLog::next() {
 		break;
 
 		case ULOG_RD_ERROR:
-			THROW_EX( HTCondorIOError, "ULOG_RD_ERROR" );
+		{
+			std::string message {"ULOG_RD_ERROR: "};;
```

Comment:
```suggestion
			std::string message {"ULOG_RD_ERROR: "};
```
Not a blocker, but I'm guessing this doesn't need two semicolons.